srcpkgs/barrier/template

```diff
@@ -11,8 +11,28 @@ short_desc="Open-source KVM software based on Synergy"
 maintainer="John <me@johnnynator.dev>"
 license="GPL-2.0-only"
 homepage="https://github.com/debauchee/barrier"
-distfiles="https://github.com/debauchee/barrier/archive/v${version}.tar.gz"
-checksum=6b92a70c5f4d625065842d133386982ec2ad1db2a809af47e46ab8ce2acd39b5
+_gmock_commit=7d33fee11ec480beae4c28ad09ca56d974140a72
+_gtest_commit=800f5422ac9d9e0ad59cd860a2ef3a679588acb4
+distfiles="https://github.com/debauchee/barrier/archive/
+v${version}.tar.gz
+ https://github.com/google/googlemock/archive/${_gmock_c
+ommit}.tar.gz
+ https://github.com/google/googletest/archive/${_gtest_c
+ommit}.tar.gz"
+checksum="259e75c150ca16d9db51870b026dc7aad56c410fa3d2f5
+fdccc19d4b6024bdc5
+ f4191fd64e8a961c06b7cc90a2bbf3774bd315254884675a900f8f3
+4fc2c4c81
+ 57bc26ac31e1bc47a12e4ae99a0f6e17506ba10f0f82ae16aa52cc7
+cc215a3b0"
+
+pre_configure() {
+        # move submodule to proper location
+        rmdir ext/gmock
+        rmdir ext/gtest
+        mv -v ../googlemock-${_gmock_commit} ext/gmock
+        mv -v ../googletest-${_gtest_commit} ext/gtest
```

Comment:
intend with tabs

Reply:
yeah ... fighting with a shallow local git copy ... will squash and push a linted version afterwards